lib/widgets/details_content.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
+import 'package:marvelapp_flutter/widgets/series_element.dart';
+import '../data/model/response_models/character.dart';
+import '../data/model/response_models/series.dart';
+
+class DetailsContent extends StatelessWidget {
+  final Character? character;
+  final List<Series>? series;
+
+  const DetailsContent(
+      {Key? key, required this.character, required this.series})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    String description = "";
+    if (character?.description != null) {
+      description = character?.description as String;
+    }
+    return Padding(
+      padding: const EdgeInsets.all(6.0),
+      child: Column(
+        verticalDirection: VerticalDirection.down,
+        children: <Widget>[
+          FadeInImage.assetNetwork(
+            placeholder: "assets/graphics/placeholder-150x150.png",
+            image:
+                "${character?.thumbnailPath}/standard_xlarge.${character?.thumbnailExtension}",
```

Comment:
Please extract private method like `String getThumbnailUrl(Character character)`.
And you probably need to check that `character  != null` at the start in line 17 because showing this page does not make sense in case there is no character.

Reply:
done